sage-ios/SAGE/SAGE/SignUpController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  SignUpController.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 10/9/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class SignUpController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate  {
```

Comment:
below this class do 

extension SignUpController: UIImage..Deleate, UINav...Delegate {
}

and then define the delegate methods in there. it separates the delegate methods in a clean way